canonical-web-and-design/maas.io
# fix: correct maas installation quickstart

## Done

Update quickstart at /install to include instructions for installing the MAAS test database, and initialising MAAS correctly.

## QA

* Run the site with `./run`
* Visit /install and verify the quickstart instructions are correct.

## Issue / Card

Fixes #524 
Fixes https://github.com/canonical-web-and-design/maas.io/issues/538

## Screenshots

![image](https://user-images.githubusercontent.com/130286/95927515-6701a900-0e1b-11eb-807f-6906fe425a90.png)
